tests/server_send.cpp

```diff
@@ -25,6 +25,8 @@
 using namespace ioremap::elliptics;
 using namespace boost::unit_test;
 
+#define ENABLE_LONG_TESTS 0
```

Comment:
I've added `DISABLE_LONG_TEST` cmake option which is disabled by default, so all long tests are enabled by default.